content/browser/cosmetic_filters_communication_impl.h

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 3.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CONTENT_BROWSER_COSMETIC_FILTERS_COMMUNICATION_IMPL_H_
+#define BRAVE_CONTENT_BROWSER_COSMETIC_FILTERS_COMMUNICATION_IMPL_H_
+
+#include <string>
+#include <vector>
+
+#include "base/memory/weak_ptr.h"
```

Comment:
headers need some cleanup, this one looks not used, some others can be moved to .cc